libompd/src/TargetValue.cpp

```diff
@@ -349,6 +349,60 @@ ompd_rc_t TValue::getRawValue(void *buf, int count) {
   return errorState.errorCode;
 }
 
+ompd_rc_t TValue::getString(const char **buf) {
```

Comment:
A comment that described the function would be helpful here. I'm assuming that "this" is holds the value of the **pointer** to a null terminated character string, buf is the callee-allocated null terminated string read from the address space, and the return code is as usual. One thing I don't understand is why you return a pointer to an empty string if the pointer is null. Isn't that an error?

Reply:
Adding the following comment for getString() in TargetValue.h.
  /**
   * Fetch a string copy from the target. "this" represents the pointer
   * that holds the value of a null terminated character string. "buf"
   * points to the destination string to be allocated and copied to.
   * Returns 'ompd_rc_error' to signify a truncated string or a target
   * read error.
   */

If/ when we incorporate the new error code, "ompd_rc_incomplete" to represent a truncated string in the standard, I will modify this comment to reflect that.

Response to the empty string question in the other comment.